### PR TITLE
Hacker News: Use search bar accessory to unify `Show HN` and `Front Page` commands

### PR DESCRIPTION
**Pull request by sxn**
## Description

- Merges the two commands into a single one, to match the native HN extension more closely.
- Adds `CHANGELOG.md` and `categories` key to manifest.

## Screencast

https://user-images.githubusercontent.com/1155589/159140390-c5fd4977-f198-42d0-bccd-28b160506a7b.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

**Comment by thomaspaulmann** [comment]
Ah, thanks for giving the extension a nice update. How about also adding the other pages to the dropdown? Maybe Ask HN and Jobs? Both of them are available on the [RSS feed](https://hnrss.github.io/).

**Comment by sxn** [comment]
good idea! I hadn’t checked what RSS feeds exist 😸  I’ve added the ones that don’t require input (like a userId), though it might be nice to have those as well in the future.

**Comment by FezVrasta** [comment]
I think the guidelines ask to use the same background for all the store screenshots.

https://developers.raycast.com/basics/prepare-an-extension-for-store#dos-and-donts

**Comment by sxn** [comment]
> I think the guidelines ask to use the same background for all the store screenshots.
> 
> https://developers.raycast.com/basics/prepare-an-extension-for-store#dos-and-donts

yup, i hadn't noticed that. My background is set to change very frequently so each screenshot ended up having a different one 😬 should be fixed now

File: extensions/hacker-news/src/StoryListItem.tsx [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+  };
+
   return (
     <List.Item
       icon={icon}
       title={props.item.title ?? "No title"}
       subtitle={props.item.creator}
-      accessoryTitle={`👍  ${points}    💬  ${comments}`}
+      accessoryIcon={hasComments ? Icon.Bubble : undefined}
+      accessoryTitle={getAccessoryTitle()}
```
Comment (thomaspaulmann) [inline extensions/hacker-news/src/StoryListItem.tsx:28]: Should we stick with just emojis until we have more flexible accessories ;-)

**Review by thomaspaulmann — APPROVED** [review]
This is great! I had a last remaining question and then we can merge it. Thanks again for advancing this extension.

File: extensions/hacker-news/src/StoryListItem.tsx (hunk 1 above)
Comment (sxn) [reply extensions/hacker-news/src/StoryListItem.tsx:28]: wanted to be closer to the native extension 😛 but i'll change this back 😉

Comment (sxn) [reply extensions/hacker-news/src/StoryListItem.tsx]: done!

**Pull request merged**